src/Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions/Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Abstractions for defining health checks in .NET applications</Description>
```

Comment:
And in a period. And/or add some "commonly used types," a la https://github.com/aspnet/EntityFrameworkCore/blob/e3e76e10ceed8a0610e05fa3f787a2470421abe5/src/EFCore/EFCore.csproj#L6-L8